packages/common/src/intl/locales/en/common.json

```
@@ -113,6 +113,7 @@
   "label.selected": "Selected",
   "label.sell": "Sell",
   "label.shipped": "Shipped",
+  "label.months": "Months",
```

```suggestion
  "label.months_one": "month",
  "label.months_other": "months",
```

to allow for plurals ( and lowercase the months 😉  )